src/Parallelization.php

```diff
@@ -373,7 +373,7 @@ protected function executeMasterProcess(InputInterface $input, OutputInterface $
                 ' ',
                 array_filter([
                     self::detectPhpExecutable(),
-                    'bin/console',
+                    realpath(getcwd().'/bin/console'),
```

Comment:
This might fail though. Could you assign it to a variable e.g. `consoleBin` and assert that the result is not false with `Assert`?

Reply:
Done, with fileExists()